src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/DataSourceUtils.java

```diff
@@ -53,7 +53,9 @@ private DataSourceUtils() {}
     private static final String  MANIFEST_SOURCE_LINE_START     = "Source:";
     private static final String  MANIFEST_ALT_SOURCE_LINE_START = "Alternate Source:";
     @VisibleForTesting
-    static final Pattern VERSION_PATTERN                        = Pattern.compile(MANIFEST_VERSION_LINE_START + "\\s+(\\d+)\\.(\\d+)\\.(\\d\\d\\d\\d)(\\d\\d)(\\d\\d)(.*)");
+    static final Pattern OLD_VERSION_PATTERN                        = Pattern.compile(MANIFEST_VERSION_LINE_START + "\\s+(\\d+)\\.(\\d+)\\.(\\d\\d\\d\\d)(\\d\\d)(\\d\\d)(.*)");
```

Comment:
Is this used anywhere?

Reply:
its in the tests... i'm somewhat remiss to get rid of the old syntax entirely since it does correspond to real releases that still exist and pretending every old version had the ref version included seems incorrect...